tests/test_consent/api/test_views.py

```diff
@@ -33,6 +34,13 @@ class TestConsentAPIViews(APITest, ConsentMixin):
     endpoint_name = 'data_sharing_consent'
     path = settings.TEST_SERVER + reverse(endpoint_name)
 
+    def setUp(self):
+        client_class = mock.patch('enterprise.models.CourseCatalogApiClient')
```

Comment:
@haikuginger Same comment here about calling this `discovery_client_class` or `catalog_client_class`.